rakelib/utils.rake

```diff
@@ -36,12 +38,12 @@ class Utils
 
   def self.podspec_as_json(file)
     file += '.podspec' unless file.include?('.podspec')
-    json, _, _ = Open3.capture3('bundle', 'exec', 'pod', 'ipc', 'spec', file)
+    json, = Open3.capture3('bundle', 'exec', 'pod', 'ipc', 'spec', file)
```

Comment:
This just seems weird to me, but I'm not that familiar with all the ruby syntax conventions, so maybe it's normal? 🤷

Reply:
Yeah, I might actually think about reverting that. This works and is valid ruby (TIL) but feels odd to me too, so maybe I'll find the corresponding rubocop rule, disable it, and revert that change to not make it too odd.

Reply:
Note that this was changed in a few places.

Reply:
57c17a41683b94c6dfba4e6171ebc04df4bb63d2